libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.h

```diff
@@ -267,10 +233,15 @@ FOUNDATION_EXTERN NSString * const kSFLoginHostChangedNotificationUpdatedHostKey
  */
 - (NSArray<SFUserAccount*> *)accountsForInstanceURL:(NSURL *)instanceURL;
 
-/** Adds a user account
- @param acct The account to be added
+/** Adds/Updates a user account
+ @param userAccount The account to be added
+ */
+- (BOOL)saveAccountForUser:(SFUserAccount *)userAccount error:(NSError **) error;
```

Comment:
Should this remain public?  Should external entities have the capacity to engage in what is fundamentally an administrative function of the user account manager?